code/script.js

```diff
@@ -0,0 +1,95 @@
+const userName = 'JohannaMJ';
+const REPOS_API_URL = `https://api.github.com/users/${userName}/repos`;
+const USER_URL = `https://api.github.com/users/${userName}`;
+const reposContainer = document.getElementById('projects');
+const profileContainer = document.getElementById('profileContainer');
+
+const userProfile = () => {
+  fetch(USER_URL)
+    .then((response) => response.json())
+    .then((data) => {
+      profileContainer.innerHTML = `
+        <div class="icon-container">
+          <a href="${data.blog}" target="blank"><img class="linkedin-icon" src="./assets/icon.png"></a>
+        </div>  
+        <div class="profile-info-container">
+          <p class="profile-text">${data.name}</p>
+          <p class="profile-text">Located in ${data.location}</p>
+          <p class="profile-text">GitHub name: ${data.login}</p>
+        </div>  
+        <div class="profile-img-container">
+          <img class="profile-img" src="${data.avatar_url}"/>
+        </div>
+      `;
+    });
+};
+
+const fetchRepos = () => {
+  fetch(REPOS_API_URL)
+    .then((response) => response.json())
+    .then((data) => {
+      const forkedRepos = data.filter(
+        (repo) => repo.fork && repo.name.startsWith('project-')
+      );
+
+      // sort function (shows latest to oldest repo)
+      forkedRepos.sort(function (oldestRepo, newestRepo) {
```

Comment:
Clear with a comment and that you named the arguments as oldestRepo and newestRepo. I just went for a & b but this describes it much better. Well done!